beacon/src/entropy.rs

```diff
@@ -36,6 +36,18 @@ impl Entropy {
         })
     }
 
+    pub fn from_data(data: Vec<u8>) -> Result<Self> {
+        let timestamp = SystemTime::now()
+            .duration_since(UNIX_EPOCH)
+            .map_err(Error::from)?
+            .as_secs() as i64;
+        Ok(Self {
+            version: 1, // marked as local
```

Comment:
what does "marked as local" mean if it's a different version of entropy than the actual local entropy i.e. `version: 0`? is this to differentiate a secondary beacon's entropy from a primary beacon's entropy but it's still traceable back to 4 bytes from the primary beaconer's OS? are these distinctions captured anywhere other than in the comment above such as the `entropy.proto`?

Reply:
yes, I wanted to distinguish the entropy source from entropy from the entropy server, I did not make a comment upstream in the proto repo yet.